freshmaker/handlers/koji/rebuild_images_on_parent_image_build.py

```diff
@@ -61,13 +72,12 @@ def handle(self, event):
             build_id=build_id
         ).first()
 
-        if found_build is not None:
-            self.set_context(found_build)
-            if found_build.event.state not in [EventState.INITIALIZED.value,
-                                               EventState.BUILDING.value]:
-                return
-            self.update_db_build_state(build_id, found_build, event)
-            self.rebuild_dependent_containers(found_build)
+        self.set_context(found_build)
+        if found_build.event.state not in [EventState.INITIALIZED.value,
+                                           EventState.BUILDING.value]:
+            return
+        self.update_db_build_state(build_id, found_build, event)
+        self.rebuild_dependent_containers(found_build)
```

Comment:
@qixiang , in the above comment @ElenaKarolinaSemanova meant we use the `found_build` variable here. So we can't remove from the `handler` method this variable.

Reply:
ah, I see, we can keep it duplicated or set `self.found_build` in `can_handle`, and then in `handle` do something like:
```
found_build = getattr(self, 'found_build', None)
if found_build is None:
    ...
```
personally I think keep this duplicated as now is simple and good enough.